vllm/engine/async_llm_engine.py

```diff
@@ -325,7 +325,12 @@ def from_engine_args(cls,
         # Create the engine configs.
         engine_configs = engine_args.create_engine_configs()
         parallel_config = engine_configs[2]
-        if parallel_config.worker_use_ray or engine_args.engine_use_ray:
+        device_config = engine_configs[4]
+
+        if device_config.device_type == "neuron":
```

Comment:
Instead of string comparison, set a `property` in device_config?

FYI, we might consider some device-specific config with `device="neuron config=(a=1, b=2, c=3)"`

Reply:
`device_type` is different from `device`. I think we can safely assume `device_type` can only be `cuda` and `neuron` for now? We can put the config string in other fields of `device_config`.

Reply:
Sure, i think we are fine to move on with this.